win32/src/winapi/kernel32/file.rs

```diff
@@ -328,3 +371,15 @@ pub fn GetFullPathNameW(
 pub fn DeleteFileA(_machine: &mut Machine, lpFileName: Option<&str>) -> bool {
     true
 }
+
+#[win32_derive::dllexport]
+pub fn GetFileAttributesA(
+    _machine: &mut Machine,
+    lpFileName: Option<&str>,
+) -> Result<FileAttribute, u32> {
+    let _file_name = lpFileName.unwrap();
+
+    // TODO: machine.host.stat
```

Comment:
There's this badly named "info" function in the file interface that you could probably replace, if that's the right place for it.  I'm not sure if GetFileAttributesA ought to be opening/closing the files it pokes at though.

Reply:
Having to open the file to run `info` is a non-starter (especially with directories! 🙂). But I imagine we'll want to add more to the opened `File` interface as well.